docs/windows.md

```diff
@@ -0,0 +1,227 @@
+
+    wins cli process run --path /k/kube-proxy/kube-proxy.exe --args "--v=4 --config=/var/lib/kube-proxy/config.conf --proxy-mode=userspace --hostname-override=$env:NODE_NAME"
+
+kind: ConfigMap
+metadata:
+  labels:
+    app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+``` 
+Set the `hostNetwork` option as true in spec of kube-proxy-windows daemonset.
+```
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    k8s-app: kube-proxy
+  name: kube-proxy-windows
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      k8s-app: kube-proxy-windows
+  template:
+    metadata:
+      labels:
+        k8s-app: kube-proxy-windows
+    spec:
+      hostNetwork: true
+```
+Then apply the `kube-proxy.yaml`.
+```
+kubectl apply -f kube-proxy.yaml
+```
+
+#### Join Windows worker Nodes
+1. (Optional, Test-Only) Install OVS provided by Antrea
+
+Antrea provides a pre-built OVS package which contains test-signed OVS kernel
+driver. If you don't have a self-signed OVS package and just want to try the
+Antrea on windows, this package can be used for testing. We also provide a help 
+script to install the OVS driver and register userspace binaries as services.
+
+Firstly, please make sure to [enable test-signed](https://docs.microsoft.com/en-us/windows-hardware/drivers/install/the-testsigning-boot-configuration-option)
+code on Windows Nodes.
+```
+Bcdedit.exe -set TESTSIGNING ON
+Restart-Computer
+```
+
+Then, install the OVS using the script.
+```
+curl.exe -LO https://raw.githubusercontent.com/vmware-tanzu/antrea/master/hack/windows/Install-OVS.ps1
+.\Install-OVS.ps1
+```
+Verify the OVS services are installed.
+```
+get-service ovsdb-server
+get-service ovs-vswitchd
+```
+
+2. Disable Windows Firewall
+
+```
+Set-NetFirewallProfile -Profile Domain,Public,Private -Enabled False
+```
+
+3. Install wins, kubelet, kubeadm and configure kubelet startup params
+
+Firstly, install wins, kubelet, kubeadm using script `PrepareNode.ps1` provided
+by kubernetes. The third component [`wins`](https://github.com/rancher/wins) is
+used to run kube-proxy and antrea-agent on Windows host inside the Windows
+container.
+```
+# Example:
+curl.exe -LO https://github.com/kubernetes-sigs/sig-windows-tools/releases/latest/download/PrepareNode.ps1
+.\PrepareNode.ps1 -KubernetesVersion v1.18.0
+```
+Then, set the Node IP used by kubelet.
+Open file `/var/lib/kubelet/kubeadm-flags.env`:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1"
+```
+Append "--node-ip=$NODE_IP" arg at the end of params. Replace `$NODE_IP` with
+the address for kubelet. It should look like:
+```
+KUBELET_KUBEADM_ARGS="--cgroup-driver= --network-plugin=cni --pod-infra-container-image=k8s.gcr.io/pause:3.1 --node-ip=$NODE_IP"```
+```
+
+4. Prepare network adapter for kubernetes service
```

Comment:
Kubernetes Service
But consider changing "for kube-proxy".

Reply:
Done